project_stock/models/project_task.py

```diff
@@ -90,12 +90,14 @@ def _compute_scrap_move_count(self):
         for item in self:
             item.scrap_count = count_data.get(item.id, 0)
 
+    @api.depends("move_ids.state")
```

Comment:
For additions and removals

```suggestion
    @api.depends("move_ids", "move_ids.state")
```